frontend/src/DiscoverMovie/Overview/DiscoverMovieOverviewInfo.js

```diff
@@ -73,7 +73,7 @@ function getInfoRowProps(row, props) {
     return {
       title: translate('Ratings'),
       iconName: icons.HEART,
-      label: `${props.ratings.tmdb.value * 10}%`
+      label: `${(props.ratings.tmdb.value * 10).toFixed(1)}%`
```

Comment:
```suggestion
      label: `${(props.ratings.tmdb.value * 10).toFixed()}%`
```

Change to .toFixed() to align with other places in the UI that have no decimal places. Good after that